pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -539,28 +540,6 @@ func TestInconsistentNameSelectorDaemonSetDoesNothing(t *testing.T) {
 	syncAndValidateDaemonSets(t, manager, ds, podControl, 0, 0)
 }
 
-func TestDSManagerNotReady(t *testing.T) {
```

Comment:
refactoring made this test obsolete, since no worker is started until the requisite caches are up to date.